lib/contracts/contribution.js

```diff
@@ -1,5 +1,20 @@
 const ethers = require('ethers');
 
+const schemas = require('kosmos-schemas');
+const tv4 = require('tv4');
+const validator = tv4.freshApi();
+
+validator.addFormat({
+  'date': function(value) {
+    const dateRegexp = /^[0-9]{4,}-[0-9]{2}-[0-9]{2}$/;
+    return dateRegexp.test(value) ? null : "A valid ISO 8601 full-date string is expected";
+  },
+  'time': function(value) {
+    const timeRegexp = /^([01][0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9]|60)(\.[0-9]+)?(([Zz])|([\+|\-]([01][0-9]|2[0-3]):[0-5][0-9]))$/;
+    return timeRegexp.test(value) ? null : "A valid ISO 8601 full-time string is expected";
+  }
+})
```

Comment:
afaik we use the same format for the proposals and a proposal once accepted creates a contribution. 
So we should probably add this also in the proposal wrapper.

Reply:
Yeah, but proposals aren't really part of MVP.

Reply:
yeah, though it might be worth to move this into a separate file that exports validator 
and then do a `const validator = require('validator')` ?

Reply:
I think it makes a lot of sense to have everything in the serializers. Still not sure how best to implement the validation in the serializer...